<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>681</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>671</width>
        <height>671</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mesh</string>
     </attribute>
     <widget class="GLDisplay" name="openGLWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>671</width>
        <height>671</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonShowObj">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveHFtoObj">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save HF to obj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadImage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from Image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>120</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Layers</string>
    </property>
    <widget class="QCheckBox" name="ck_br">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bedrock</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_sable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sable</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_eau">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Eau</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_montagne">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Montagne</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_total">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="box_nx">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_ny">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_hauteurmax">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_hauteurmin">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>320</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>nx/ny</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>320</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hauteur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>55</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dim.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_dimentionMin">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_dimentionMax">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>350</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_change_nx_ny">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change nx/ny</string>
    </property>
   </widget>
  </widget>
  <action name="actionSubmenu">
   <property name="text">
    <string>submenu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDisplay</class>
   <extends>QWidget</extends>
   <header location="global">gldisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
